khelo/lib/ui/flow/matches/add_match/add_match_screen.dart

```diff
@@ -424,366 +297,91 @@ class _AddMatchScreenState extends ConsumerState<AddMatchScreen> {
-                  ),
-                  _matchOfficialsCell(
-                    context: context,
-                    notifier: notifier,
-                    state: state,
-                    title:
-                        context.l10n.add_match_officials_live_streamers_title,
-                    image: "assets/images/ic_streamer.png",
-                  ),
-                  _matchOfficialsCell(
-                    context: context,
-                    notifier: notifier,
-                    state: state,
-                    title: context.l10n.add_match_officials_others_title,
-                    image: "assets/images/ic_other_official.png",
-                  ),
-                  const SizedBox(
-                    width: 16,
-                  ),
-                ],
-              )),
-        ),
-      ],
-    );
-  }
-
-  Widget _matchOfficialsCell({
-    required BuildContext context,
-    required AddMatchViewNotifier notifier,
-    required AddMatchViewState state,
-    required String title,
-    required String image,
-  }) {
-    return OnTapScale(
-      onTap: () async {
-        final officials =
-            await AppRoute.addMatchOfficials(officials: state.officials)
-                .push<List<Officials>>(context);
-        if (officials != null && context.mounted) {
-          notifier.setOfficials(officials);
-        }
-      },
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.center,
-        children: [
-          Expanded(
-            child: Container(
-              height: 110,
-              width: 110,
-              alignment: Alignment.center,
-              padding: const EdgeInsets.all(8),
-              decoration: BoxDecoration(
-                border: Border.all(
-                    color: context.colorScheme.containerNormalOnSurface),
-                shape: BoxShape.circle,
-                color: context.colorScheme.containerLowOnSurface,
+        SectionTitle(title: context.l10n.add_match_match_schedule_title),
+        Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16.0),
+          child: Row(
+            children: [
+              Expanded(
+                child: DropDownSelectionButton(
+                    buttonText:
+                        state.matchTime.format(context, DateFormatType.date),
+                    headerImage: Assets.images.icCalendar,
+                    headerText: context.l10n.add_match_date_title,
+                    buttonPlaceHolder: context.l10n.add_match_date_title,
+                    onButtonTap: () {
+                      _selectDate(context, notifier, state);
+                    }),
               ),
-              child: Image.asset(
-                image,
-                fit: BoxFit.cover,
-                color: context.colorScheme.textDisabled,
+              const SizedBox(width: 16),
+              Expanded(
+                child: DropDownSelectionButton(
```

Comment:
I think we need to rename this button because this button not work as drodownbutton.

Reply:
although it doesn't work as dropdown button, this button's UI is similar to dropDown button also Note that this button is used in **EditProfileScreen**  as well where it's UI is like dropdown button.

![dropDown](https://github.com/canopas/khelo/assets/122426509/efaa87d9-c60c-4a69-9821-fa2addfd04aa)

Reply:
But here date and time selection button are not similar to dropdown i think you can use outlined button.

Reply:
DropDownSelectionButton already handles outline border as well as header text, header icon, title, placeholder, trailing icon and Tap event, then why do we need to use outlined button? I understand that the name of component doesn't align with the use case in this screen but I don't think that changing widget is required. however you can suggest a name for this widget if you have any.